spec/lib/mailchimp_spec.rb

```diff
@@ -72,15 +75,57 @@
 																method: 'POST',
 																path: 'lists/list_id/members',
 																body: an_instance_of(String)
-														},
-														{
-																method: 'DELETE',
-																path: 'lists/list_id/members/on_mailchimp'
 														}
 													])
 		end
   end
 
+	# 1st option -- drip_email_list, nonprofit, user 
+	# describe 'sign up nonprofit user' do 
+
+	# 	it 'signs up nonprofit user' do 
+
+	# 		expect(Mailchimp).to receive(:drip_email_list).with(email_list).and_return(ret_val)
+
+	# 		result = Mailchimp.create_nonprofit_user(email_list, true)
+	# 		expect(result).to match([
+	# 													{
+	# 															method: 'PUT',
+	# 															path: '',
+	# 															body: an_instance_of(String)
+	# 													}
+	# 												])
+	# 	end 
+
+	# 	it 'does not sign up nonprofit user' do 
+
+	# 		expect(Mailchimp).to receive(:drip_email_list).with(email_list).and_return(ret_val)
+
+	# 		result = Mailchimp.create_nonprofit_user(email_list, true)
+	# 		expect(result).to match([
+	# 													{
+	# 															method: 'PUT',
+	# 															path: '',
+	# 															body: an_instance_of(String)
+	# 													}
+	# 												])
+	# 	end 
+	# end 
+
+	# 2nd option
+	describe '.signs up nonprofit user' do 
+
+		it 'signs up nonprofit user' do 
+			user = create(:user, :drip_email_list_base, nonprofit: np)
+			expect(Mailchimp.get_drip_email_list(nonprofit.id, user.id)).to eq ['test_user@email.com']
+		end
+
+		it 'does not sign up nonprofit user' do 
+			user = create(:user, :drip_email_list_base, nonprofit: np)
+			expect(Mailchimp.get_drip_email_list(nonprofit.id, user.id)).to be_empty
+		end
+	end 
+
```

Comment:
Hold off on this until you finish the previous three tasks in https://github.com/CommitChange/tix/issues/4094